ui/app/controllers/clients/client/index.js

```diff
@@ -338,13 +338,4 @@ export default class ClientController extends Controller.extend(
     }
   }
   // #endregion metadata
-
-  // Show an additional column in the Allocations table, but only if any of them have actions.
-  get allocationsHaveActions() {
-    return this.sortedAllocations.any((a) => {
-      return a.states.any((s) => {
-        return s.get('task')?.actions.length;
-      });
-    });
-  }
```

Comment:
Until this PR, I conditionally showed an "Actions" column on the node index page. Problem is, you might have a bunch of jobs, some of which have actions, and you'd have to parse through all of them to determine if that column should be shown.

So I just decided heck, let's show the column all the time.